src/components/Home/UpgradeNotice.tsx

```diff
@@ -30,21 +30,29 @@ const UpgradeWrapperDiv = styled.div`
   }
 `;
 
+interface IUpgradeNoticeProps {
+  canClusterUpgrade: boolean;
+  loadingReleases: boolean;
+  onClick?: () => void;
```

Comment:
It's common practice to have optional parameters at the end of the interface declaration